erts/doc/src/erl_cmd.xml

```diff
@@ -422,6 +422,22 @@
           </p>
         </warning>
       </item>
+      <tag><c><![CDATA[-no_epmd]]></c></tag>
+      <item>
+        <p>Specifies that the distributed node does not need
+          <seecom marker="epmd">epmd</seecom> at all.</p>
+        <p>This option ensures that the Erlang runtime system does not
+          start <seecom marker="epmd">epmd</seecom> and does not start
+          the <seeerl marker="kernel:erl_epmd">erl_epmd</seeerl> process
+          for distribution either.</p>
+        <p>This only applies if Erlang is started as a distributed node
```

Comment:
```suggestion
        <p>This option only works if Erlang is started as a distributed node
```

Reply:
Change included in the updated version, thanks for your review.